CONTRIBUTING.md

```diff
@@ -8,152 +8,104 @@
-- Autogenerate the changelog (https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes), then sort the changes alphabetically; if all change names are prefixed with the component name, this will ensure changes are grouped by component. Sorting can be done using the `sort` utility available in MacOS and Linux distros; Clean up unnecessary commits (e.g automated commits like pre-commit.ci)
-- Write a summary at the top of the release page. It should include sections:
-  - Major features include
-  - Backwards incompatible changes
-  - Removal of features
-- Include Install instructions (link to https://github.com/vmware/versatile-data-kit/wiki/Installation).
-- Make sure you have clicked the "Create a discussion for this release" button before publishing the new release
-- Post a tweet on the official VDK Twitter account, and on slack, announcing the new release and linking to it.
-
-
-## Ask for Help
-
-The best way to reach us with a question when contributing is to ask on:
-
-* The original GitHub issue
-* [Our Slack channel](https://cloud-native.slack.com/archives/C033PSLKCPR)
-* Join our [Community Meetings](https://github.com/vmware/versatile-data-kit/wiki/Community-Meeting-and-Open-Discussion-Notes)
+Before submitting your pull request, review the following:
+- Check if your code changes pass both code linting checks and unit tests.
+- Ensure your commit messages are descriptive. Be sure to include any related GitHub issue references in the commit message. See [Basic writing and formatting syntax](https://guides.github.com/features/mastering-markdown/) for guidelines on syntax formatting.
+- Check the commit and commit messages to ensure they are free from spelling and grammar errors.
+- Use clear commit titles and commit descriptions for generating changelog release notes.
+
+## Submit a pull request
+- All changes must be submitted to the main branch using pull requests.
+- Any change must go on a feature branch or on a fork.
+- Pipeline must pass before merging.
+- Code commits must be broken down into small self-contained units.
+- Commit messages must follow the template in [git-commit-template.txt](https://github.com/vmware/versatile-data-kit/blob/main/support/git-commit-template.txt). See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit) as a guideline to write commit messages.
+- The change must abide by the [Versatile Data Kit Coding Standard](https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard).
+- Each change is a subject to two code reviews and approvals before it is merged.
+
+We prefer to maintain a straight branch history by rebasing before merging. Fast-forward merges should not create merge conflicts.
+
+# Development Workflow
+A typical development workflow has the following process:
+- Create a topic branch from where you want to base your work, naming your branch according to our naming convention. For example `person/`.
```

Comment:
I think person/< github-username >/< feature-name > is important to keep here for the naming convention